src/main/resources/helptexts/dates.html

```diff
@@ -4,7 +4,7 @@
     than the date on which the digital files were completed (select "issued" from the menu) or the
     date on which entries were changed (select "modified").</p>
 
-<p><strong>Note:</strong> The date on which the digital files were closed should be recorded at 'Dateset created'.</p>
+<p><strong>Note:</strong> The date on which the digital files were closed should be recorded at 'Dataset created'.</p>
```

Comment:
By the way is "at" correct here? Sounds funny. Maybe "as" or "in" was intended?

Reply:
I think it should be 'in' indeed.

```suggestion
<p><strong>Note:</strong> The date on which the digital files were closed should be recorded in 'Dataset created'.</p>
```